src/Npgsql/Util/ResettableCancellationTokenSource.cs

```diff
@@ -128,8 +128,8 @@ public void ResetCts()
     /// </remarks>
     public void Stop()
     {
-        _cts.CancelAfter(InfiniteTimeSpan);
         _registration.Dispose();
+        _cts.CancelAfter(InfiniteTimeSpan);
```

Comment:
Was there a particular race condition that prompted this change?

Reply:
Nah, it's just so it works the same way Reset does.